src/ua-daemon.c

```diff
@@ -381,6 +381,7 @@ static void bus_acquired_cb(GDBusConnection *connection, const gchar *name,
 static void name_lost_cb(GDBusConnection *connection, const gchar *name,
                          gpointer user_data) {
   UaDaemon *self = user_data;
+  g_message("Name '%s' lost", name);
```

Comment:
Should this still be here? g_debug perhaps if so?

Reply:
Well, can be still somewhat informative to have it in journal in case something goes bad, but I'm fine with both.

Reply:
It's fine if it's intentional, it just looked like it might have been a mistake.